frontend/src/utils/notifications.ts

```diff
@@ -0,0 +1,5 @@
+// The default `autoHideDuration` used by the package notistack is 5000 ms.
+// This value is good for form success or error messages, as it gives the
+// users enough time to read. For more simple operations, like clicking on
+// copy button, a quicker feedback appropriate.
+export const SNACKBAR_DYNAMIC_FEEDBACK_MS = 1800;
```

Comment:
In another open PR, I see camelcase utils constants. Should we have a uniform way for all constants? Are there different types of constants for which the two styles make sense?

Reply:
I'm not sure if there is a consensus in JavaScript, but I know that on the "Google JavaScript Style Guide" constants are uppercase and snake case.

See: https://google.github.io/styleguide/jsguide.html#naming-constant-names